tutorials/ui/bbcode_in_richtextlabel.rst

```diff
@@ -9,54 +9,183 @@ Introduction
 :ref:`class_Label` nodes are great for displaying basic text, but they have limitations.
 If you want to change the color of the text, or its alignment, you can only do that to
 the entire label. You can't make a part of the text have another color, or have a part
-of the text centered. To get around these limitations you would use a :ref:`class_RichTextLabel`.
+of the text centered. To get around these limitations, you would use a :ref:`class_RichTextLabel`.
 
 :ref:`class_RichTextLabel` allows for complex formatting of text using a markup syntax or
 the built-in API. It uses BBCodes for the markup syntax, a system of tags that designate
 formatting rules for a part of the text. You may be familiar with them if you ever used
 forums (also known as `bulletin boards`, hence the "BB" in "BBCode").
 
-Note that the BBCode tags can also be used, to some extent, in the XML source of
+Note that the BBCode tags can also be used to some extent in the XML source of
 the class reference. For more information, see :ref:`doc_class_reference_primer`.
 
+.. seealso::
+
+    You can see how BBCode in RichTextLabel works in action using the
+    `Rich Text Label with BBCode demo project <https://github.com/godotengine/godot-demo-projects/tree/master/gui/rich_text_bbcode>`__.
+
 Using BBCode
 ------------
 
-By default :ref:`class_RichTextLabel` functions exactly the same as the normal label. It
-has the :ref:`property_text <class_RichTextLabel_property_text>` property, which you can
-edit to have uniformly formatted text. To be able to use BBCodes and rich text formatting
+By default, :ref:`class_RichTextLabel` functions exactly the same as a normal :ref:`class_Label`.
```

Comment:
```suggestion
By default, :ref:`class_RichTextLabel` functions like a normal :ref:`class_Label`.
```
There are some differences between default `RichTextLabel` and `Label`, such as the minimum size and scrollbar.